module-system/sov-modules-macros/src/module_call_json_schema.rs

```diff
@@ -0,0 +1,26 @@
+use syn::DeriveInput;
+
+pub fn derive_module_call_json_schema(
+    input: DeriveInput,
+) -> Result<proc_macro::TokenStream, syn::Error> {
+    let DeriveInput {
+        ident, generics, ..
+    } = input;
+
+    let (impl_generics, type_generics, where_clause) = generics.split_for_impl();
+
+    let tokens = quote::quote! {
+        use ::schemars::JsonSchema;
+
+        impl #impl_generics ::sov_modules_api::ModuleCallJsonSchema for #ident #type_generics #where_clause {
+            fn json_schema() -> String {
```

Comment:
Let's return `::std::string::String` (macro hygiene)

Reply:
Good catch, fixed in https://github.com/Sovereign-Labs/sovereign-sdk/pull/517/commits/04a37bff3a172c5e4c13d07738c532178c3f5c76.